validator/validator-main.protoascii

```diff
@@ -5321,6 +5320,39 @@ tags: {
   }
 }
 
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "IMG"
+  spec_name: 'I-AMPHTML-INTRINSIC-SIZER'
+  mandatory_parent: "I-AMPHTML-SIZER"
+  attrs: {
+    name: "class"
+    value_regex: "i-amphtml-intrinsic-sizer"
+    mandatory: true
+  }
+  attrs: {
+    name: "role"
+    value: "presentation"
+    mandatory: true
+  }
+  attrs: {
+    name: "alt"
+    value: ""
```

Comment:
Is an explicitly empty `alt` attribute really part of the sizer transformation? Seems unnecessary.

Reply:
https://davidwalsh.name/accessibility-tip-empty-alt-attributes

Reply:
oh. interesting.